parser/parser_unix.go

```diff
@@ -18,6 +18,15 @@ func getPkgPath(fname string, isDir bool) (string, error) {
 		fname = path.Join(pwd, fname)
 	}
 
+	gopath := os.Getenv("GOPATH")
+	if gopath == "" {
+		var err error
+		gopath, err = getDefaultGoPath()
+		if err != nil {
+			return "", fmt.Errorf("cannot determine GOPATH: %s", err)
+		}
+	}
+
 	for _, p := range strings.Split(os.Getenv("GOPATH"), ":") {
```

Comment:
@jen20 have you forgot to actually use local `gopath` here, or I just miss something?

/cc @rvasily

Reply:
Yea, it unused.
@jen20 can you fix thi in separate PR ?